examples/foam/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+TODO_DIR=$PWD/`dirname $0`
+FOAM_DIR=$TODO_DIR/node_modules/foam-framework
+OUT_DIR=$TODO_DIR/build
+
+rm -rf $OUT_DIR
+mkdir $OUT_DIR
+node $FOAM_DIR/tools/foam.js foam.build.BuildApp targetPath=$TODO_DIR/build \
+  controller=com.todomvc.Controller \
+  htmlFileName=junk.html \
+  extraModels=foam.ui.BooleanView,foam.ui.DetailView,foam.ui.ActionButton \
+  extraClassPaths=$TODO_DIR/js
+
+rm $OUT_DIR/junk.html
```

Comment:
Only if we want to be able to rebuild the `build/foam.js` script that is being run. I don't know what the policies are here. It isn't necessary if we just want the code checked in and don't care about rebuilding it.